File: Rdpzsd.Models/Dtos/Rdpzsd/Lot/PersonLotDto.cs
using Rdpzsd.Models.Models.Nomenclatures;
using Rdpzsd.Models.Models.Rdpzsd;

namespace Rdpzsd.Models.Dtos.Rdpzsd.Lot
{
    public class PersonLotDto
    {
        public PersonLot PersonLot { get; set; }
        public string PersonBasicNames { get; set; }
        public string PersonBasicNamesAlt { get; set; }
        public Country Citizenship { get; set; }
        public Country SecondCitizenship { get; set; }
        public bool HasActualStudentOrDoctoral { get; set; }
        public bool? PersonSecondaryFromRso { get; set; }
    }
}

File: WebApplication/src/rdpzsd/dtos/lot/person-lot.dto.ts
import { Country } from "src/nomenclatures/models/settlement/country.model";
import { PersonLot } from "src/rdpzsd/models/person-lot.model";

export class PersonLotDto {
    personLot: PersonLot;
    personBasicNames: string;
    personBasicNamesAlt: string;
    citizenship: Country;
    secondCitizenship: Country;
    hasActualStudentOrDoctoral: boolean;
    personSecondaryFromRso: boolean;
}